index.js

```diff
@@ -298,7 +312,9 @@ const run = async userOptions => {
       } else if (options.asyncJs) {
         await asyncJs({ page });
       }
-      const filePath = path.join(destinationDir, route);
+      const publicPath = options.publicPath.slice(0, -1); // remove trailing slash
```

Comment:
isn't it supposed to be done with regexp `/\/$/`

Reply:
In this case the last char will always be slash but i agree it's easier to read the intention with a regexp, i will update 👍